packages/core/src/lambda/remoteDebugging/ldkLayers.ts

```diff
@@ -31,7 +31,7 @@ export const regionToAccount: RegionAccountMapping = {
 }
 
 // Global layer version
-const globalLayerVersion = 1
+const globalLayerVersion = 2
```

Comment:
Do we always need to update this version manually. Can we instead use `$LATEST`?